src/views/pages/website/home/elements/Cards.tsx

```diff
@@ -5,14 +5,34 @@ import { Owner, Repository } from "../../../../../model";
 import { Audience } from "../../../../Audience";
 
 export interface CardsProps {
-  owner: Owner;
-  repository: Repository;
-  audience: Audience;
+  owner?: Owner;
+  repository?: Repository;
+  audience?: Audience;
   action?: string;
   to?: To;
+  isLoading?: boolean;
 }
 
 export function Cards(props: CardsProps) {
+  if (props.isLoading) {
+    return (
```

Comment:
There is way too much copy-paste in this section. Please remove the copy-paste element. If we want to change the design we should not have to change it twice